varint.js

```diff
@@ -87,3 +85,9 @@ function readUnsignedVarIntFrom(buffer, offset) {
     var got = offset - start;
     return ReadResult.shortError(got + 1, got, start, offset);
 }
+
+function countBits(n) {
```

Comment:
I just ran a benchmark with n from 1 to 10^7, bit operation won.
Math.log(n)/Math.log(2)
real	0m0.415s
user	0m0.406s
sys	0m0.010s

n.toString(2).length:
real	0m2.847s
user	0m2.840s
sys	0m0.018s

countBits(n):
real	0m0.192s
user	0m0.184s
sys	0m0.008s